app/cypress/e2e/cif/project-revision/edit-project-revision.cy.js

```diff
@@ -151,22 +151,19 @@ describe("when editing a project, the project page", () => {
     cy.findByText("Amendment 2").should("be.visible"); // ensure the submit page has loaded
 
     // edit teimp
-
     cy.findByRole("button", { name: /Emissions Intensity Report/i }).click();
     cy.findByText(/Edit emissions intensity report/i).click();
-    cy.findByRole("button", {
-      name: /Add Emissions Intensity Report/i,
-    }).click();
-    cy.addEmissionIntensityReport(
+    cy.addOrEditEmissionIntensityReport(
```

Comment:
I think we additionally need a few assertions down by line 192ish to check these diffs like we're checking the other diffs